### PR TITLE
Add Orlando hometown file

### PR DESCRIPTION
**Pull request by githubteacher**
This pull request adds the Orlando hometown file to the project.

@shahnj could you please take a 👀 ? 🙏 🙇 🆘

### CODE REVIEW

**Comment by shahnj** [comment]
looks good 👍 -- Let's merge this

File: hectorsector-orlando.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#  Orlando, FL
+
```
Comment (shahnj) [inline hectorsector-orlando.md]: @githubteacher Good, but which part of Orlando? That would give me a better idea when I visit you! 🥇

**Review by shahnj — CHANGES_REQUESTED** [review]

**Comment by JohnGedik** [comment]
I am not sure if it looks good.

**Review by shahnj — APPROVED** [review]
@githubteacher - That's great - I am planning to stay in Oakland, FL during Christmas. 👍

**Pull request merged**